### DISCUSSION TITLE
我刚答完所有题，即时显示的得分是不是错的啊？

### DISCUSSION THREAD

**Discussion by wangchuanyes** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#31] [Q&A]
我刚答完所有题，去除小作文题显示得分57分。是不是显示错了啊，是不是还是显示的我上周的分数啊？
题目解析我看了好多次完全理解了才答的题，而且模拟答题我也做了两遍没错误。

**Comment by luminosara** [comment]
同样是57分，最后一页题目里显示是只有选择题成绩，应该是填空成绩还没出？

但按照每道题8分，送分题1分算，57+3*8=81也不够

**Comment by zhouym233** [comment]
折腾了半天整了个66分，目前看到最高有74分的。

**Comment by 603712046** [comment]
+1啊，我也是看着今天的题目刚答完就显示57分，不可能有填错的啊，我专门看了好几遍😂
